msu/systems/serialization/serialization_system.nut

```diff
@@ -42,24 +44,28 @@
 		}
 		if (_flags == null) _flags = ::World.Flags;
 
-		local inEmulator = ::MSU.Class.DeserializationEmulator(_mod, _id, _flags);
+		local inEmulator = ::MSU.Class.FlagDeserializationEmulator(_mod, _id, _flags, ::MSU.System.Serialization.DeserializationMetaData);
```

Comment:
The manual passing of the `DeserializationMetaData` is now unnecessary as we have added handling for null argument in the constructor of the `FlagDeserializationEmulator`.